roles/manage-aws-infra/tasks/main.yml

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
Can we please split this into multiple files - i.e.: `stop_start_aws.yml`, `terminate_aws.yml`, etc.

Reply:
Sure, no problem